Vision and Autonomous: Boilerplate FRCVision

KhanSimeoni opened the pull request:
Inital code for the FRC vision system.

KhanSimeoni committed:
this code broke

KenwoodFox committed:
Remove mess

KenwoodFox committed:
Fix api url

KenwoodFox committed:
Fix formatting

KenwoodFox committed:
Fix again

KenwoodFox committed:
Clean getencoder error

KenwoodFox committed:
Better logging

KenwoodFox committed:
Bump readme

KenwoodFox committed:
Add hyperlink+notes to readme

KenwoodFox committed:
fixup! Add hyperlink+notes to readme

KenwoodFox committed:
Not the right ratio, but better than before

KenwoodFox committed:
try something

KenwoodFox committed:
Fixup

KenwoodFox committed:
Fix all, done

KenwoodFox committed:
Add comments to workflows

KenwoodFox committed:
refactor yaml

KhanSimeoni committed:
kinda works now

KhanSimeoni committed:
better I thinkq

KhanSimeoni committed:
still not working almost

KenwoodFox committed:
Add more networktables publushing, fix inital bugs with radian conversion

KenwoodFox committed:
Git blame

KenwoodFox committed:
More comments

KhanSimeoni committed:
Forgot to commit my comments last night, looks like we both commented most of drivetrain. I added the comments that wern't covered by yours - Khan, to Joe

KenwoodFox committed:
Fork off from khan and work on implementing wpilib's on-rio PID loops

KenwoodFox committed:
Merge branch 'main' into bugfix/swerve

KenwoodFox committed:
Merge branch 'main' into bugfix/swerve

KhanSimeoni committed:
Give the vision pi a static ip

KenwoodFox reviewed (changes requested):
Id like to see more with networktables before this gets merged.

Kredcool committed:
Bugfix9 (#23)

Co-authored-by: KenwoodFox <kenwood364@gmail.com>
This has some issues we need to work out.

KenwoodFox committed:
Merge branch 'main' into bugfix/swerve

KenwoodFox committed:
Fix broken merge with

tbohlen committed:
Fixes #24, unspolling on startup and steer ration
This adds some additional print statements and a test button command to
make testing a bit easier.

KhanSimeoni committed:
Merge remote-tracking branch 'origin/bugfix/swerve' into feature/aivision

Kredcool committed:
made base vision file

Kredcool committed:
added __init__

KenwoodFox reviewed (changes requested):
Sorta getting there.

KhanSimeoni committed:
Merge branch 'feature/aivision' of https://github.com/FRC-1721/1721-RapidReact into feature/aivision

KhanSimeoni committed:
a

KhanSimeoni committed:
more

KhanSimeoni committed:
limelight config

KenwoodFox committed:
Clean limelight artifacts

KenwoodFox committed:
Bump gitignore

KenwoodFox committed:
Merge branch 'main' into feature/aivision

KenwoodFox committed:
Need more than just cscore

KenwoodFox committed:
Update dependencies

KhanSimeoni committed:
limelight follower

KenwoodFox commented:
Ill fix this soon

KenwoodFox committed:
Not sure why that was there

KenwoodFox reviewed (changes requested):
@KhanSimeoni i saw you changing all that code, did you push it up?

KenwoodFox committed:
Cleanup ip list

KhanSimeoni committed:
Merge branch 'feature/aivision' of https://github.com/FRC-1721/1721-RapidReact into feature/aivision

KhanSimeoni committed:
we gonna break

KenwoodFox commented on rio/robotcontainer.py:
```diff
@@ -100,6 +101,10 @@ def configureButtonBindings(self):
             Kicker(self.yoke)
         )
 
+        commands2.button.JoystickButton(self.driverController, 5).whenPressed(
+            LimeAuto(self.drivetrain)
+        )
+
```
This looks good :)

KhanSimeoni committed:
couple bad networktable lookups,fixed now
